lib/utilities/monitoringHandler.js

```diff
@@ -0,0 +1,65 @@
+const errors = require('arsenal');
+const client = require('prom-client');
+
+function writeResponse(res, error, log, results, cb) {
+    let statusCode = 200;
+    if (error) {
+        if (Number.isInteger(error.code)) {
+            statusCode = error.code;
+        } else {
+            statusCode = 500;
+        }
+    }
+    res.writeHead(statusCode, { 'Content-Type': 'application/json' });
+    res.write(JSON.stringify(results));
+    res.end(() => {
+        cb(error, results);
+    });
+}
+
+
+function routeHandler(req, res, log, cb) {
+    if (req.method !== 'GET') {
+        return cb(errors.BadRequest, []);
+    }
+    client.collectDefaultMetrics();
+    const promMetrics = client.register.metrics();
+    const contentLen = Buffer.byteLength(promMetrics, 'utf8');
+    res.setHeader('content-length', contentLen);
+    res.setHeader('content-type', 'application/json');
```

Comment:
Wrong MIME-type for the response. There's a constant defined I believe in prom-client, see https://github.com/siimon/prom-client#utilites

Reply:
If I check the type, its string.
So it should be text/plain.
Is that okay?

Reply:
> The content-type prometheus expects is also exported as a constant, both on the register and from the main file of this project, called contentType.